packages/manager/apps/key-management-service/src/pages/serviceKey/serviceKey.page.tsx

```diff
@@ -51,10 +55,20 @@ export default function Key() {
 
   const navigate = useNavigate();
 
-  const kms = okms?.data;
-  const kmsKey = serviceKey?.data;
+  if (isLoading || isLoadingOkms) return <Loading />;
 
-  if (isLoading) return <Loading />;
+  if (okmsError)
+    return (
+      <ErrorBanner
+        error={okmsError.response}
+        onRedirectHome={() => navigate(ROUTES_URLS.listing)}
+        onReloadPage={() =>
+          queryClient.refetchQueries({
+            queryKey: getOkmsServiceKeyResourceQueryKey({ okmsId, keyId }),
+          })
+        }
+      />
+    );
```

Comment:
Maybe is better create ErrorBannerPage as sub component to avoid duplicate this component ? Because only error is different.

Or use :
```ts
        error={okmsError?.response ?? error.response}
```

Just as suggestion.

And a warning : 
It's normal it's not different queryKey ?